src/calculateRentalCost.js

```diff
@@ -5,6 +5,27 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRent = 40;
+  const shortTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  const totalRentPrice = days * dailyRent;
+
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return totalRentPrice;
+  }
+
+  if (days >= 3 && days < 7) {
+    return totalRentPrice - shortTermDiscount;
+  }
```

Comment:
Since you have already returned `totalRentPrice` for `days < 3`, you don't need to check `days >= 3` again here. It's guaranteed that `days` is at least `3` at this point, so you can simplify the condition to just check if `days < 7`.